automation/test.sh

```diff
@@ -392,7 +392,7 @@ if [[ -z ${KUBEVIRT_E2E_FOCUS} && -z ${KUBEVIRT_E2E_SKIP} ]]; then
   elif [[ $TARGET =~ (cnao|multus) ]]; then
     label_filter='(Multus,Networking,VMIlifecycle,Expose,Macvtap)'
   elif [[ $TARGET =~ sig-network ]]; then
-    label_filter='(sig-network)'
+    label_filter='(sig-network,netCustomBindingPlugins)'
```

Comment:
> Only clusters that deploy the network binding plugin CNI should run the test.

Where is this checked?
I don't understand the benefit of this label, it seems that sig-network will always run those tests. Any other target will skip it anyway.